tests/raftstore/util.rs

```diff
@@ -89,6 +89,7 @@ pub fn new_store_cfg() -> Config {
         max_leader_missing_duration: Duration::from_secs(3),
         report_region_flow_interval: 100, // 100ms
         raft_store_max_leader_lease: TimeDuration::milliseconds(25 * 10),
+        use_sst_file_snapshot: true,
```

Comment:
It needs a feature like "test suite" with "setup/teardown", which could be used to setup different config for different tests/rerun.
It seems there is no easy way to do something similar in cargo test. Any better idea is appreciated.